Tasks/app-store-release/app-store-release.ts

```diff
@@ -128,7 +130,7 @@ async function run() {
         process.env['PATH'] = process.env['PATH'] + ':' + gemCache + path.sep + 'bin';
 
         // Ensure there's exactly one ipa before installing fastlane tools
```

Comment:
Can we update this comment?

Reply:
I will update the comments after merging.